tests/data_context/datasource/test_data_context_datasource_runtime_data_connector_sqlalchemy_execution_engine.py

```diff
@@ -97,31 +96,6 @@ def test_get_batch_successful_specification_pandas_engine_named_asset_two_batch_
     assert batch_2.batch_definition.batch_identifiers == batch_identifiers
 
 
-def test_get_batch_ambiguous_parameter_sqlalchemy_engine(
```

Comment:
@donaldheppner I do not intuitively see why these tests are being removed. Could you please help with a justification? Thank you.

Reply:
These tests check the RuntimeBatchRequest constructor for the same logic that is tested other places and in test_batch.py -- the actual assertions here have nothing to do with Spark.